tests/smoke-tests/test-eth-parent-hash-bad-block-fix.ts

```diff
@@ -0,0 +1,76 @@
+import "@moonbeam-network/api-augment";
+import { ApiDecoration } from "@polkadot/api/types";
+import chalk from "chalk";
+import { expect } from "chai";
+import { describeSmokeSuite } from "../util/setup-smoke-tests";
+
+const debug = require("debug")("smoke:ethereum-contract");
+
+describeSmokeSuite("S570", `RPC Eth ParentHash`, async function (context, testIt) {
+  let atBlockNumber: number = 0;
+  let previousBlockNumber: number = 0;
+  let apiAt: ApiDecoration<"promise"> = null;
+  let apiAtPrevious: ApiDecoration<"promise"> = null;
+
+  before("configure api at block", async function () {
+    this.timeout(6_000_000); // 30 minutes
```

Comment:
This is incorrect. 6mio millisecond is 100 minutes, half an hour is `1_800_000`

You can instead use `THIRTY_MINS` constant from `../util/constants` which is more readable